backend/routes.go

```diff
@@ -25,7 +26,9 @@ func (s *server) handleRoutes() error {
 
 	router.HandleFunc("/users/", s.handleAddUser()).Methods("POST")
 	router.HandleFunc("/users/{id}", s.handleGetUser()).Methods("GET")
-	router.HandleFunc("/articles", s.handleGetArticles()).Methods("GET")
+	//Adding route param...
+	router.HandleFunc("/articles/", s.handleGetArticles()).Methods("GET")
+	router.HandleFunc("/articles/{count}", s.handleGetArticles()).Methods("GET")
```

Comment:
Sweet! I wonder if we can use optional query parameters instead of a path.. i.e something like this https://stackoverflow.com/questions/43379942/how-to-have-an-optional-query-in-get-request-using-gorilla-mux

I know I said to use `vars` in the issue, but this seems a little neater since they are optional (and otherwise we would return all articles if it's not provided)

That way, you can have something like this in the handler,
```
v := r.URL.Query()
count := v.Get("count")
// ... We can add the strconv
```

And we would avoid setup an extra `handleFunc` up here